Runtime.md

```diff
@@ -0,0 +1,91 @@
+This guide will describe how GamesRunnr runs upkeep functions and how FSPAnalyzer is used to schedule them in GameStartr projects.
+
+### Table of Contents
+1. [GamesRunnr](#gamesrunnr)
+    1. [Games](#games)
+    2. [Interval and Speed](#interval-and-speed)
+    3. [Pause and Play](#pause-and-play)
+2. [FPSAnalyzer](#fpsanalyzer)
+
+# GamesRunnr
+
+GamesRunnr is a module for running a series of callbacks on timed intervals.
+These callbacks deal with maintaining the state of the game.
+An example being a callback to redraw the canvas.
+These callbacks separately are referred to as games.
+
+## Games
+
+`upkeep` is the biggest part of GamesRunnr.
+This function deals with running all the stored games.
+
+`upkeepTimed` is a utility function for `upkeep` that times and returns the amount of time it takes to run all the games.
+
+```typescript
+let totalTime: number = GamesRunnr.upkeepTimed();
+console.log(`It took `${totalTime}` ms.`);
+```
+
+This is used to help calculate `upkeepNext` which is a number reference to the next upkeep scheduled by `upkeepScheduler`.
+`upkeepScheduler` is a custom function that takes in a handler for another function and sets it to run after a delay.
+
+```typescript
+GamesRunner.upkeepScheduler(() => console.log("This ran after 200 ms."), 200);
```

Comment:
Don't document using private variables. `upkeepScheduler` and `upkeepNext` are intentionally private so that nobody touches them.